core/common/src/main/scala/org/typelevel/otel4s/KindTransformer.scala

```diff
@@ -54,14 +55,25 @@ trait KindTransformer[F[_], G[_]] {
 
   /** Lifts a natural transformation from `F` to `F` into a natural
     * transformation from `G` to `G`.
+    *
+    * @note
+    *   Implementors SHOULD NOT override this method; the only reason it is not
+    *   final is for optimization of the identity case.
     */
-  final def liftFunctionK(f: F ~> F): G ~> G =
+  def liftFunctionK(f: F ~> F): G ~> G =
     new (G ~> G) {
       def apply[A](ga: G[A]): G[A] = limitedMapK(ga)(f)
     }
 }
 
 object KindTransformer {
+  implicit def identity[F[_]]: KindTransformer[F, F] =
```

Comment:
Could the name clash with the `identity` function from the std package? This concern is valid only for the explicit import, e.g. `import KindTransformer._`, which would rarely happen. 

Should we call it `id` similarly to the `FunctionK.id`? 

I'm fine with both options

Reply:
`id` sounds good. thanks for pointing that out 👍